rosdep/base.yaml

```
@@ -2196,6 +2196,9 @@ imagemagick:
   gentoo: [virtual/imagemagick-tools]
   nixos: [imagemagick]
   ubuntu: [imagemagick]
+intel-opencl-icd:
+  debian: [intel-opencl-icd]
```

This package is not available in Debian Buster so the distribution version cannot be elided.

```suggestion
  debian:
    '*': [intel-opencl-icd]
    buster: null
```